pkg/kfapp/ossm/ossm_manifests.go

```diff
@@ -50,21 +59,23 @@ func (o *OssmInstaller) processManifests() error {
 	if err := o.SyncCache(); err != nil {
 		return internalError(err)
 	}
+	ControlPlaneDir := "templates/control-plane"
```

Comment:
Did the `const` commit ge lost?

Reply:
It's present in the branch but not in the PR - https://github.com/maistra/opendatahub-operator/tree/embed-templates No idea how that's possible.

Reply:
I see it now that I've pushed another commit (from your suggestion). Odd.